tests/e2e/e2e_test.go

```diff
@@ -54,7 +55,7 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 
 func (s *IntegrationTestSuite) TestBankTokenTransfer() {
 	s.Run("send_photon_between_accounts", func() {
-
+		s.T().Skip()
```

Comment:
why is this being skipped now?

Reply:
Ah I left that in when I was trying to isolate a single test I'll bring it back